lib/src/ui/pages/profile/profile_page.dart

```diff
@@ -1,12 +1,61 @@
 import 'package:allthenews/generated/l10n.dart';
+import 'package:allthenews/src/di/injector.dart';
+import 'package:allthenews/src/ui/pages/profile/auth/authentication_page.dart';
 import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import 'auth/authorization_notifier.dart';
+
+abstract class _Constants {
+  static const fontFamily = 'Chomsky';
+}
+
+class ProfilePage extends StatefulWidget {
+  @override
+  _ProfilePageState createState() => _ProfilePageState();
+}
+
+class _ProfilePageState extends State<ProfilePage> {
+  final AuthorizationNotifier _authorizationNotifier = inject<AuthorizationNotifier>();
+
+  @override
+  void initState() {
+    super.initState();
+    _authorizationNotifier.getCurrentUser();
+  }
 
-class ProfilePage extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return Container(
-      alignment: Alignment.center,
-      child: Text(Strings.of(context).profile),
-    );
+    return ChangeNotifierProvider.value(
+        value: _authorizationNotifier,
+        builder: (providerContext, child) {
+          final viewState =
+              providerContext.select((AuthorizationNotifier notifier) => notifier.state);
+
+          if (viewState.user == null) {
+            return AuthenticationPage();
+          } else {
+            return Center(
+              child: Column(
+                mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                children: [
+                  Text(_authorizationNotifier.state.user?.email),
+                  Text(_authorizationNotifier.state.user?.displayName),
+                  Text("User uid: ${_authorizationNotifier.state.user?.uid}"),
```

Comment:
do stringsów :)

Reply:
docelowo myślę, że nie powinniśmy tej informacji wyświetlać na UI, dałem bardziej na potrzeby debugowania, proponuję, że dodam TODOsa na usunięcie tego przy okazji implementacji widoku profilu?

Reply:
zawsze można ograć ładnie, tak żeby wyświetlać tylko w devowej wersji, tj. notifier miałby informację czy wyświetlać czy nie na podstawie Environment :)

Reply:
raczej mało przydatne info, mail jest i tak unikatowy, w sumie to już teraz wywalę